MatrixSDK/MXSession.h

```diff
@@ -923,7 +923,17 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
 
  @return the MXRoom instance.
  */
-- (MXRoom  *)roomWithRoomId:(NSString*)roomId;
+- (MXRoom *)roomWithRoomId:(NSString*)roomId;
+
+/**
+ Get the MXRoom instance of a room.
+ Create it if does not exist yet.
```

Comment:
Can we just improve this doc to point out the room won't be created in Matrix layer? It might be misinterpreted by SDK users.